tests/ert/unit_tests/storage/test_storage_migration.py

```diff
@@ -146,18 +145,6 @@ def test_that_storage_matches(
 
         # We need to normalize some irrelevant details:
         experiment.parameter_configuration["PORO"].mask_file = ""
-        if version.parse(ert_version).major == 5:
-            # In this version we were not saving the full parameter
-            # configuration, so it had to be recreated by what was
-            # in ErtConfig at the time of migration, hence the new
-            # path
-            experiment.parameter_configuration[
-                "BPR"
-            ].template_file = experiment.parameter_configuration[
-                "BPR"
-            ].template_file.replace(
-                str(tmp_path), "/home/eivind/Projects/ert/test-data"
-            )
```

Comment:
Should this be removed?

Reply:
yes, since `parameter_configuration` does not contain template_file anymore.

Reply:
I've included assert for templates_configuration